cypress/integration/percy/article.web.spec.js

```diff
@@ -15,7 +15,7 @@ describe('For WEB', function() {
             // Prevent the Privacy consent banner from obscuring snapshots
             cy.setCookie('GU_TK', 'true');
             // Fix the location to UK (for edition)
-            cy.setCookie('GU_EDITION', 'UK');
+            cy.setCookie('GU_EDITION', 'EU');
```

Comment:
I don't think there's a EU edition, though probably just works as Int is default. The edition cookie value is `INT` though.

Reply:
When Tom and I tested this last night the cookie was getting set to 'EU' for UK. But checking Percy again, I'm still seeing International edition being picked 😑